### ISSUE TITLE
Quick Allocation

### ISSUE THREAD

**Issue by malfynnction** [envelope-zero/frontend#514]
Dashboard should have a "quick allocation" button, opening a popup with the different options (as soon as they are implemented in the backend)
<img width="359" alt="image" src="https://user-images.githubusercontent.com/24372341/203857933-cce19b24-f4bd-4188-944d-da7244d6a0a4.png">
<img width="354" alt="image" src="https://user-images.githubusercontent.com/24372341/203857952-50b9eb09-0588-4ac1-a3ed-785859529520.png">

**Comment by malfynnction** [comment]
https://github.com/envelope-zero/backend/issues/440

**Comment by malfynnction** [comment]
@morremeyer You will need to help me with user friendly translations here, because tbh I don't fully understand what `ALLOCATE_LAST_MONTH_BUDGET` and `ALLOCATE_LAST_MONTH_SPEND` mean 🤔

**Comment by malfynnction** [comment]
Are "Average spent last year" and "set all to 0" still modes you would like to have? If so, let me know when you've implemented them in the backend! :)

**Comment by morremeyer** [comment]
Yes, they are. I'll add the waiting label again and will let you know.

**Comment by morremeyer** [comment]
Depends on https://github.com/envelope-zero/backend/issues/294
